src/types/game_engine.rs

```diff
@@ -0,0 +1,101 @@
+use std::mem;
+
+use super::{IScriptable, Native, Ref, ScriptClass, Type};
+use crate::raw::root::RED4ext as red;
+use crate::types::WeakRef;
+use crate::{NativeRepr, VoidPtr};
+
+#[derive(Default)]
+#[repr(transparent)]
+pub struct GameInstance(red::ScriptGameInstance);
```

Comment:
```suggestion
pub struct ScriptGameInstance(red::ScriptGameInstance);
```
I'd follow the red4ext names here

Reply:
renamed in 3308d8c.